Source/Charts/Utils/ViewPortHandler.swift

```diff
@@ -54,11 +54,7 @@ open class ViewPortHandler: NSObject
     
     /// offset that allows the chart to be dragged over its bounds on the x-axis
     fileprivate var _transOffsetY = CGFloat(0.0)
-    
-    public override init()
-    {
-    }
-    
+
```

Comment:
empty line?

Reply:
This one should be here. The rest are now fixed